packages/core/src/modules/ledger/services/IndyLedgerService.ts

```diff
@@ -109,6 +109,8 @@ export class IndyLedgerService {
     try {
       this.logger.debug(`Set endpoints for did '${did}' on ledger '${pool.id}'`, endpoints)
 
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
```

Comment:
This should work if you have the latest @types/indy-sdk version

```suggestion
```

Reply:
ah yesssss. forgot to update the deps